wordpress-mobile/WordPress-Android
# Support forum:  remove feature flag

Removes Support Forum Feature Config as it is no longer required.

Fixes #18816 

| WordPress | Jetpack |
|--------|--------|
|![Screenshot_20230724_195301](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/d76e044b-0edd-46a8-89ba-6cc15536611c) | ![Screenshot_20230724_195344](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/1fea2184-7433-49e8-9d4f-8a0da7af54c0)| 


To test:

- Launch WP app
- Go to Help screen (Me -> Help)
- `Verify` the help screen is as shown in first image above, and there is no `Contact support` option
- Tap on `Community forums`
- `Verify` it goes to WP Mobile App Forum on the web
- Logout
- Try the above steps from Help on Login screen
- `Verify` it goes to Help screen, and `Community forums` goes to forum on the web
===============================================================
- Launch JP app
- Navigate to Help screen (Me -> Help)
- `Verify` help screen looks as second image above
- Tap on Contact support
- `Verify` it opens Zendesk chat (after confirming email, if not done already)


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Existing tests, and manual testing

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
